### PR TITLE
i have edit the media responsive css style and make that title font 4…

### PR DESCRIPTION
**Pull request by WillyFerry**
…0px so that will be not get bug like before

# sabesansathananthan.github.io

GitHub Flow PR Image

Thank you so much for opening a pull request! :smiley:

Please provide the following information where possible to help the pull request reviewer merge your PR quicker!

## Relevant Project Issue Numbers :hash:

*Add any related issue numbers from the GitHub project. Append the `#` symbol before issue numbers.*

>e.g. #58

## Involved Project Members :bust_in_silhouette:

*Tag any GitHub usernames involved or directly responsible for reviewing the proposed changes. Append the `@` symbol before usernames.*

>e.g @Charlotteis
 
## An Explanation of Your Changes :speech_balloon:

*Your explanation goes here!*

## Any Screenshots of Your Changes :camera:
 
*Your screenshots go here! Use Markdown Syntax to embed images.*

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -3260,3 +3260,9 @@ h6 {
     top: 0;
     left: 0;
   }
+@media only screen and (max-width: 600px) {
+  h1.name{
+    font-size:40px;
+    overflow:hidden;
+  }
+}
```
Comment (sabesansathananthan) [inline css/style.css]: shall we add the follow code inside the response . css file

File: blog.html [hunk 2]
```diff
@@ -94,7 +94,7 @@ <h2>BLOG</h2>
                                    
                                             <div class="jumbotron text-center">
                                                 <div id="logo"></div>
-                                                <h1 class="display-4 mb-3">@sabesan96's Posts</h1>
+                                                <h1 class="display-4 mb-3 name" style="overflow: hidden;">@sabesan96's Posts</h1>
```
Comment (sabesansathananthan) [inline blog.html]: want we need the `style="overflow: hidden;"` part ?

Comment (WillyFerry) [reply blog.html]: no ,, i forget to delete that ,, sorry sir , you can delete that or must i delete it?

File: css/style.css (hunk 1 above)
Comment (WillyFerry) [reply css/style.css]: if you wanna add it to response .css file , just replace that code to .css file, but dont forget to delete the code from style.css if you have replace it to .css file

File: blog.html (hunk 2 above)
Comment (WillyFerry) [reply blog.html]: style="overflow:hidden;" part have i add to the style css , but i forget to delete the inline css ,, sorry about that

Comment (sabesansathananthan) [reply blog.html]: No problem. make another PR and I suggest you to participate in this hackathon.

https://hacktoberfest.digitalocean.com/details#quality-standards

**Comment by sabesansathananthan** [comment]
get latest changes from master and change the suggestions. push it

File: blog.html (hunk 2 above)
Comment (WillyFerry) [reply blog.html]: please check it

**Pull request merged**